src/aks-preview/azext_aks_preview/tests/latest/test_managed_cluster_decorator.py

```diff
@@ -8161,6 +8161,68 @@ def test_update_supportPlan(self):
         normalClusterCalculated = noopDecorator3.update_k8s_support_plan(normalCluster)
         self.assertEqual(normalClusterCalculated, normalCluster)
 
+    def test_mc_get_node_init_taints(self):
```

Comment:
Could you add some unit tests to `test_agentpool_decorator.py` or move some of these there?

Reply:
Agent pool level operations (nodepool add/update) with node initialization taints are currently blocked (pending some dependency on cluster autoscaler, to be changed for GA of this feature).

Would it be okay to not include tests in this case, since we are not supporting the agentpool level operations by themselves?